backend/WebUI/api_webui.go

```diff
@@ -411,9 +411,14 @@ func JWT(email, userId, tenantId string) string {
 	mu.Lock()
 
 	if jwtKey == "" {
-		rand.Seed(time.Now().UnixNano())
 		for i := 0; i < 256; i++ {
-			jwtKey += string(rune(rand.Intn(128)))
+			randomBytes := make([]byte, 128)
+			_, err := rand.Read(randomBytes)
+			if err != nil {
+				logger.ProcLog.Warnln("Generate JWT Private Key error.")
```

Comment:
Does this mean that if creating a random key fails, the signature is done with an empty string as key?

Reply:
Yes, but I think that it would not happened. This err checking scope is added to pass the golangci-lint.